src/main/java/gov/nih/nci/evs/api/fhir/R4/ValueSetProviderR4.java

```diff
@@ -193,6 +195,19 @@ public ValueSet expandImplicit(
             OperationOutcome.IssueType.EXCEPTION,
             500);
       }
+
+      // If properties are indicated, retrieve the concept with all potentially
+      // needed info
+      IncludeParam includeParam = new IncludeParam("minimal");
```

Comment:
This declaration is useless (and confusing).  Where you use "includeParam = " below, just declare it there.  And the same thing in expandInstance().

IncludeParam includeParam = new IncludeParam(String.join(",", includeList));